File: 2 Factory/src/MSSQL.java
public class MSSQL implements IDB {
    @Override
    public void selectData() {
        System.out.println("MSSQL veritabanından veri okundu.");
    }

    @Override
    public void insertData() {
        System.out.println("MSSQL veritabanına veri yazıldı.");
    }
}

File: 2 Factory/chsarp/MSSQL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Factory_csharp
{
    public class MSSQL : IDB
    {
        public void SelectData()
        {
            Console.WriteLine("MSSQL veritabanından veri okundu.");
        }


        public void InsertData()
        {
            Console.WriteLine("MSSQL veritabanına veri yazıldı.");
        }
    }
}
